### PR TITLE
add poi-label gazetteer

### PR DESCRIPTION
**Pull request by aparlato**
Adds poi-label gazetteer.

cc @nickidlugash

### CODE REVIEW

File: mapbox-streets/poi-label.json [hunk 1]
```diff
@@ -0,0 +1,462 @@
+ {
+    "type": "FeatureCollection",
+    "name": "poi-label",
```
Comment (nickidlugash) [inline mapbox-streets/poi-label.json]: Perhaps make the name more specific?

File: mapbox-streets/poi-label.json [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+                "type": "Point",
+                "coordinates": [
+                    -122.463,
+                    37.799
+                ]
+            },
+            "properties": {
+                "place_name": "Presidio and San Francisco National Cemetery, USA",
+                "zoom": 16,
+                "description": "Two low scalerank POIs next to one another surrounded by higher scalerank POIs.",
```
Comment (nickidlugash) [inline mapbox-streets/poi-label.json]: scalerank --> sizerank?

File: mapbox-streets/poi-label.json [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+                "type": "Point",
+                "coordinates": [
+                    -98.457,
+                    29.358
+                ]
+            },
+            "properties": {
+                "place_name": "San Antonio State Hospital, San Antonio TX, USA",
+                "zoom": 15,
+                "description": "San Antonio State Hospital.",
```
Comment (nickidlugash) [inline mapbox-streets/poi-label.json]: The description field is optional – maybe don't include one if it's not going to have any more information that isn't already in the name? Ideally though I think we use this field: https://github.com/mapbox/gazetteer/blob/gazetteer-3/CONTRIBUTING.md#description

**Comment by aparlato** [comment]
@nickidlugash I think I've addressed all of your comments now!

> Many (but not all) of the highlights in this gazetteer include the id data field – is there a particular reason you wanted to include this for some of the features?

This was a leftover since I started this gazetteer before writing in the code that deleted the `id` field. We need the `id` field while creating the entry, but once the entry is created we delete this now in carto-cam since it's no longer necessary.

**Comment by nickidlugash** [comment]
@aparlato LGTM!

**Pull request merged**